src/Dropdown.js

```diff
@@ -101,14 +110,18 @@ class Dropdown extends React.Component {
     return this.menuRef.current;
   }
 
+  getToggle() {
+    return this.toggleRef.current;
+  }
```

Comment:
I don't think we need this method as on line `119` we are not using this.

Reply:
We are using that on line `154`, so added to `119` as well.